### PR TITLE
Document the compatability issue of auto mTLS and exlucdeInboundPort.

### PR DESCRIPTION
**Pull request by incfly**
Fixed https://github.com/istio/istio/issues/30311

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: content/en/faq/security/automtls-exlucde-port.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Does Auto mTLS work with "excludeInboundPorts" annotations?
+weight: 80
+---
+
+No. When `traffic.sidecar.istio.io/excludeInboundPorts` is used on the server workloads, Istio still
```
Comment (smawson) [inline content/en/faq/security/automtls-exlucde-port.md:6]:
```suggestion
No. When `traffic.sidecar.istio.io/excludeInboundPorts` is used on server workloads, Istio still
```

File: content/en/faq/security/automtls-exlucde-port.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Does Auto mTLS work with "excludeInboundPorts" annotations?
```
Comment (smawson) [inline content/en/faq/security/automtls-exlucde-port.md:2]:
```suggestion
title: Does Auto mTLS work with the "excludeInboundPorts" annotations?
```

File: content/en/faq/security/automtls-exlucde-port.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+weight: 80
+---
+
+No. When `traffic.sidecar.istio.io/excludeInboundPorts` is used on the server workloads, Istio still
+configures the client Envoy to send mutual TLS by default. To change that, you need to configure
+a Destination Rule with mutual TLS mode `DISABLE` to make client send plain text to those ports.
```
Comment (smawson) [inline content/en/faq/security/automtls-exlucde-port.md:8]:
```suggestion
a Destination Rule with mutual TLS mode set to `DISABLE` to have clients send plain text to those ports.
```

File: content/en/faq/security/automtls-exlucde-port.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Does Auto mTLS work with the "excludeInboundPorts" annotations?
```
Comment (rvennam) [inline content/en/faq/security/automtls-exlucde-port.md:2]: This title is confusing in my opinion. What does "work with" mean?

I suggest something along the lines of... Does Auto mutual TLS exclude ports set using "excludeInboundPorts" annotation?

Comment (craigbox) [reply content/en/faq/security/automtls-exlucde-port.md:2]: Please also rename the file from `automtls-exlucde-port.md` to fix the typo.

Comment (incfly) [reply content/en/faq/security/automtls-exlucde-port.md]: done.

**Comment by incfly** [comment]
@rvennam @craigbox PTAL, thanks.

**Review by rvennam — APPROVED** [review]

**Pull request merged**